libnavui-dropin/api/current.txt

```diff
@@ -245,6 +245,7 @@ package com.mapbox.navigation.dropin {
     method public com.mapbox.maps.plugin.annotation.generated.PointAnnotationOptions? getDestinationMarkerAnnotationOptions();
     method public Integer? getEndNavigationButtonStyle();
     method public Integer? getInfoPanelBackground();
+    method public Float? getInfoPanelGuidelineMaxPosPercent();
```

Comment:
looks unrelated. there must be an issue with the api check

Reply:
Someone forgot to update the files, when adding these APIs. But better late than never, right?

Reply:
I rebased on `main`, this is no longer present in my PR.